src/rime/lever/deployment_tasks.cc

```diff
@@ -277,6 +277,15 @@ static bool RemoveVersionSuffix(string* version, const string& suffix) {
     return false;
 }
 
+static bool CheckConfigFileExists(const fs::path& source_config_path,
+                                  const fs::path& dest_config_path) {
+  if (!fs::exists(source_config_path) ||
+      !fs::exists(dest_config_path)) {
+    return false;
```

Comment:
It's not required that config files exist in both paths.

Reply:
Sorry I don't get it.
How to start running RimeApi without loading default.yaml?

Reply:
You can't; then don't.
This change wouldn't help. It doesn't stop you from running without configuration.
This, `ConfigFileUpdate`, is not for `default.yaml` alone but for updating any configuration file.
Your changes would break valid uses cases.